service/controller/v13/templates/cloudformation/guest/outputs.go

```diff
@@ -2,6 +2,8 @@ package guest
 
 const Outputs = `{{define "outputs"}}
 Outputs:
+  HostedZoneNameServers:
+    Value: !Join [ ',', !GetAtt 'HostedZone.NameServers' ]
```

Comment:
Yeah this should be aligned with the approach we have in `GetCurrentState`. We fetch all of these already and have a lot of logic to guarantee these values down the road in order to inject them where needed. The current implementation hijacks this so lets please align it a bit. We can also talk and go through the code together if you want.

Reply:
I have todo here https://github.com/giantswarm/giantswarm/issues/3314
If it happens to be too much work, I'll convert that to be an issue.